cursorless-talon/src/modifiers/relative_scope.py

```diff
@@ -5,13 +5,76 @@
 mod = Module()
 
 
-@mod.capture(rule="(previous | next) <user.cursorless_scope_type>")
+@mod.capture(rule="previous | next")
+def cursorless_relative_direction(m) -> str:
+    """Previous/next"""
+    return "backward" if m[0] == "previous" else "forward"
+
+
+@mod.capture(rule="backward")
+def cursorless_relative_backward() -> str:
+    """Backward"""
+    return "backward"
+
+
+@mod.capture(
+    rule="[<user.ordinals_small>] <user.cursorless_relative_direction> <user.cursorless_scope_type>"
```

Comment:
I don't believe we have any tests for `"third next"`

Reply:
No tests are still on my to do list that's why the task isn't completed in this pr :)

Reply:
Right sounds good.  Fwiw I'm pretty sure those are the only tests necessary; I believe I added tests for everything else